hana/extract_hana_package.sls

```diff
@@ -0,0 +1,47 @@
+{%- from "hana/map.jinja" import hana with context -%}
+
+{% if hana.hana_archive_file is defined %}
+{% set hana_package = hana.hana_archive_file %}
+{% set hana_extract_dest = hana.hana_extract_dir %}
```

Comment:
Why have you changed from `hana_extract_dir` to `hana_extract_dest`? If you prefer, change the default name in `defaults.yml` and pillar file, now that you have changed. Put the best one from the beginning.
I would just keep the same name to make the code easier to read.
The same for `hana_packages`.
It's only cosmetics, and It's not a big deal, just curiosity.

Reply:
I think `hana_extract_dest` is better and aligns with the `dest` parameter for extraction function s in salt, unix. I will use it for all locations.

Reply:
Thinking again, I prefer to use `hana_extract_dir` because this variable is used for extraction and later for installation, and for installation having `dest` word as input variable is confusing